be/src/udf/java/java_udf.cpp

```diff
@@ -482,7 +477,7 @@ Status ClassLoader::init() {
     _get_class = env->GetMethodID((jclass)_clazz.handle(), "findClass", "(Ljava/lang/String;)Ljava/lang/Class;");
     _get_call_stub =
             env->GetMethodID((jclass)_clazz.handle(), "generateCallStubV",
-                             "(Ljava/lang/String;Ljava/lang/Class;Ljava/lang/reflect/Method;)Ljava/lang/Class;");
+                             "(Ljava/lang/String;Ljava/lang/Class;Ljava/lang/reflect/Method;I)Ljava/lang/Class;");
```

Comment:
What's effect of the change of this line?

Reply:
for UDFClassLoader.java. I change the method arguments for generateCallStubV.
To distinguish between UDF and UDAF callstub
```java
        if (genType == SINGLE_BATCH_UPDATE) {
            bytes = CallStubGenerator.generateCallStubV(clazz, method);
        } else if (genType == BATCH_EVALUATE) {
            bytes = CallStubGenerator.generateScalarCallStub(clazz, method);
        } else {
            throw new UnsupportedOperationException("Unsupported generate stub type:" + genType);
        }
```